Makefile

```diff
@@ -116,29 +122,26 @@ copy:
 	   docker cp $$ID:$(BASE)/kernel/artifacts/ . && \
 	   docker rm -v $$ID
 
-cartesi-linux-config:
-	cp $(KERNEL_CONFIG) ./cartesi-linux-config
+$(DEP_DIR):
+	mkdir dep
+
+$(KERNEL_SRCPATH): | $(DEP_DIR)
+	wget -O $@ https://github.com/cartesi/linux/archive/refs/tags/v$(KERNEL_VERSION).tar.gz
```

Comment:
Forcing `tags` instead of any `refs` brings no security benefit while it prevents us from using CI for development branches. We'll have to keep hacking this links to build in CI. So I disagree with this in favor of what was here before the latest push-force.

It was like this: `wget -O $@ https://github.com/cartesi/linux/archive/refs/heads/$(KERNEL_BRANCH).tar.gz`.

Same for opensbi link.

Reply:
My example was poorly written. It should have been, not branch directly:

```
OPENSBI_REF=tags/v1.3.1-ctsi-1
```
or
```
OPENSBI_REF=heads/ports/opensbi-1.3.1
```

I agree, we can get back to this at a later point.